### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by rosemaina**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,8 @@
+bucketenv/
+__pycache__/
+*.coverage
+*.pyc
+htmlcov/
+.vscode/
+*.noseids
+.noseids
```
Comment (Lemmah) [inline .gitignore:8]: Awesome .gitignore file...

File: .noseids [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+(S'/home/rossie/Documents/bucket_list/app/models/tests/user_tests.py'
+g23
+S'Testuser.test_register_user'
+tp27
+sI14
+(S'/home/rossie/Documents/bucket_list/app/models/tests/user_tests.py'
+g23
+S'Testuser.test_user_existence'
+tp28
+ss.
```
Comment (Lemmah) [inline .noseids:100]: Shouldn't this one be ignored too?

**Review by Lemmah — APPROVED** [review]
Good work! Congratulations...

File: .noseids (hunk 2 above)
Comment (rosemaina) [reply .noseids:100]: I have tried.
Even included it in the .gitignore file but it is not getting ignored

**Pull request merged**